dqc/api/properties.py

```diff
@@ -460,6 +483,28 @@ def _equadrupole(qc: BaseQCCalc) -> torch.Tensor:
 
     return quadrupole + ion_quadrupole
 
+@memoize_method
+def _optimal_geometry(qc: BaseQCCalc) -> torch.Tensor:
+    # calculate the optimal geometry
+    system = qc.get_system()
+    atompos = system.atompos
+
+    # check if the atompos requires grad
+    _check_differentiability(atompos, "atom positions", "hessian")
```

Comment:
The error message here will be `"Differentiable tensor atom positions is required to calculate the hessian"`, which I think it is a bit misleading, i.e. do we calculate the hessian? Maybe the last argument would be better be `"optimal geometry"` or something like that

Reply:
fixed